app/graphql/types/user_type.rb

```diff
@@ -0,0 +1,8 @@
+module Types
+  class UserType < Types::BaseObject
+    field :id, ID, null: false
```

Comment:
We will never have a nil ID in our database, but we have said that name, email, birthdate _could_ potentially be nil. We can easily change this.

Reply:
I think this works.  Great work!

Reply:
One thing we might want to consider, the convention around dates in the US vs other countries. I don't think that impacts how we store the data, as a datetime, but may affect how we display it.

Reply:
@krogowsk531 @taylorjohnson141 @aemcdonald what is the best way for us to send you dates? String, integer, none of the above?

Reply:
@dhalverson You're right, JavaScript can be really finicky about dates https://stackoverflow.com/questions/7556591/is-the-javascript-date-object-always-one-day-off and we might have to take that in account on the frontend if we end up creating dates . @leahriffell  I think a String would be easiest. Also is backend going to be making the dates at all? If everything is stored in the correct String format we want to display neither frontend nor backend should have to but sometimes the dates are easier to interact with if they are in the form of a date class.

Reply:
We built our tables to store dates as opposed to strings. I think we will need to deal with dates on the backend to any time-related math so it makes sense to store those values as dates. We could store them as strings but I think we would need to convert them to a date for calculations anyways. There's also the fact that we may be dealing with "week numbers" more often than with actual dates! Which we should probably send to FE as integers. So:

- String for dates (we can decide on a format later)
- Integers for week numbers

How does that sound?

Reply:
That sounds good to me! I also saw this type of graphql format that we could look into: ` GraphQL::Types::ISO8601DateTime`. 

_This scalar takes DateTimes and transmits them as strings, using ISO 8601 format._ 
So they would still be sent as strings but maybe the formatting is more consistent?